app/assets/hub/kong-inc/ai-proxy/anthropic.yaml

```diff
@@ -0,0 +1,21 @@
+description: |
+  This configuration enables AI Proxy with Anthropic.
+
+prereqs: 
+  - A route with `paths[]=~/anthropic-chat$` configured. Apply the plugin to this route.
+
+custom_values: 
+  - config.auth.header_value
```

Comment:
Do we need the `config.` prefix? Are all custom values implicitly in the config?

Reply:
I think that's a question for @fabianrbz - what would be better?

Reply:
I don't think we need it, they should be implicitly in the config.
However, I don't know who and how this API will be consumed, as long as it's clear to everyone I'm ok with dropping it.